app/brokers/(3-techincal-docs)/javascript-sdk/params/page.mdx

```diff
@@ -51,13 +51,21 @@ Live price slippage can be specified by using the `livePriceSlippageTolerancePer
 property. It must be a number between 0 and 100. The default behavior is to use the recommended
 value from the quote but can be overridden or set to `false` to disable live price protection.
 If the quote does not have a recommended live price slippage tolerance, this setting is not
-available for the asset pair and should not be set.
+available for the asset pair and should NOT be set.
 
 If either of the slippage tolerance properties are invalidated, the swap will be refunded to
 the specified refund address. Refunds are subject to a broadcast fee on the source chain to pay for the
 transactions sent by the Chainflip protocol. The boost or deposit fee paid will not
 not be refunded.
 
+<Callout type="warning">
+  If overriding live price slippage protection - the tolerance should account
+  for network fees and broker fees. For example, if the network fee is 0.1% and
+  the broker fee is 1.5% - the live price slippage tolerance should be set to at
+  least 0.25% to avoid refunds. The recommended value returned from the quote
```

Comment:
Out of curiosity, how do you get this 0.25% based on the 0.1 and 1.5? (or is it a typo and should it be 0.1 and 0.15?)